integration-tests/jest/1-run-workflow.integration-test.js

```diff
@@ -7,4 +7,4 @@ const { registerTest } = require('./jest-utils')
 const { testRunWorkflow } = require('../tests/run-workflow')
 
 
-registerTest(testRunWorkflow)
+registerTest(testRunWorkflow, 1)
```

Comment:
This one really don't fail very often. Occasionally, Cromwell and/or Rawls are overloaded and the workflow gets stuck in "queued" or "submitted" or "running" and doesn't finish in time. A re-run might fix it, but might just apply more load.

I think I've also seen an occasional failure where it's trying to navigate to the workspace where the workspace list has been filtered, the workspace is visible, but for some reason doesn't get clicked. I bet we could fix this. (This could probably also affect most other tests, but for some reason, I feel like it has mostly affected this one.)